.github/workflows/extraction.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Extract Info
+on:
+  schedule:
+    # runs at 6 AM UTC
+    - cron:  '0 6 * * *'
```

Comment:
Runs at 6 am UTC which is 9 AM Israel time. this hour was chosen by the last file uploaded on the supermarkets websites